### ISSUE TITLE
Green does not support Jade and Liquid

### ISSUE THREAD

**Issue by nakoshi-satamoto** [Blockstream/green_qt#118]
Blockstream Green lacks support for Jade and Liquid. I was under the impression that Jade was a Liquid hardware wallet, is this not the case?

On the latest firmware version 1.0.28 (no bluetooth) and on the latest version of Green 2.0.2 when trying to create a Liquid account in Green for Jade it returns the graphical error attached.

![green-LN-Jade-01](https://github.com/Blockstream/green_qt/assets/161100720/2544b5c2-188f-4df6-9360-4e857f1deb45)
![green-LN-Jade-02](https://github.com/Blockstream/green_qt/assets/161100720/8528506b-283f-459b-9861-363b79598062)

**Comment by nakoshi-satamoto** [comment]
I was attempting singlesig. Log file did not show anything of much use.

**Comment by nakoshi-satamoto** [comment]
It works with bitcoin but not liquid.

**Comment by nakoshi-satamoto** [comment]
If it matters at all, i was using a proxy to local tor instance. But connections are fine as I was able to test setting up the jade and saw connections being successful in the log file.

**Comment by nakoshi-satamoto** [comment]
Disabling tor and proxy makes no difference. I also tested with the bluetooth enabled firmware and also with the prevoius version of Green. Green simply just does not support Jade and Liquid.

Does Jade act as a Liquid hardware wallet as advertised? If so what app should I be using?

**Comment by delta1** [comment]
this seems to be the only relevant log i could find 

```
[2024-03-26 09:26:09.736736] [gdk:debug] 09.736 INFO - GDKRUST_call_session load_store output "Ok(Null)"
[2024-03-26 09:26:09.736736] [gdk:debug] 09.736 INFO - GDKRUST_call_session handle_call get_master_blinding_key input "Null"
[2024-03-26 09:26:09.736736] [gdk:debug] 09.736 INFO - GDKRUST_call_session get_master_blinding_key output "redacted"
[2024-03-26 09:26:09.756756] [gdk:debug] 09.756 INFO - GDKRUST_call_session handle_call get_master_blinding_key input "Null"
[2024-03-26 09:26:09.756756] [gdk:debug] 09.756 INFO - GDKRUST_call_session get_master_blinding_key output "redacted"
[2024-03-26 09:26:12.372372] [app:info] "[ViewsModule] closeViewWithID:  viewId = 1418334964_1711437883828592767"
[2024-03-26 09:26:16.131131] [app:info] "[Countly][endSession]"
[2024-03-26 09:26:16.131131] [gdk:debug] 16.131 INFO - GDKRUST_call_session handle_call disconnect input "Null"
[2024-03-26 09:26:16.131131] [gdk:debug] 16.131 INFO - push notification: Object {"event": String("network"), "network": Object {"current_state": String("disconnected"), "next_state": String("disconnected"), "wait_ms": Number(0)}}
[2024-03-26 09:26:16.131131] [gdk:debug] 16.131 INFO - push notification: Object {"event": String("network"), "network": Object {"current_state": String("disconnected"), "next_state": String("disconnected"), "wait_ms": Number(0)}}
[2024-03-26 09:26:16.131131] [gdk:info] destroying singlesig session 0x7e056c001580
[2024-03-26 09:26:16.131131] [gdk:info] destroying singlesig session 0x7e056c001580
[2024-03-26 09:26:16.131131] [gdk:debug] 16.131 INFO - GDKRUST_call_session disconnect output "Ok(Null)"
04] [gdk:debug] 16.604 INFO - latest serialization hash matches, no need to flush
[2024-03-26 09:26:16.609609] [gdk:debug] 16.609 INFO - latest serialization hash matches, no need to flush
[2024-03-26 09:26:16.609609] [gdk:debug] 16.609 INFO - push notification: Object {"event": String("network"), "network": Object {"current_state": String("disconnected"), "next_state": String("disconnected"), "wait_ms": Number(0)}}
[2024-03-26 09:26:16.609609] [gdk:debug] 16.609 INFO - GDKRUST_call_session disconnect output "Ok(Null)"
[2024-03-26 09:26:16.609609] [gdk:debug] 16.609 INFO - GDKRUST_call_session handle_call disconnect input "Null"
[2024-03-26 09:26:16.609609] [gdk:debug] 16.609 INFO - GDKRUST_call_session disconnect output "Ok(Null)"
[2024-03-26 09:26:16.609609] [gdk:info] destroying singlesig session 0x5c7745b44a60
[2024-03-26 09:26:16.610610] [gdk:info] reconnect_hint: hint:disconnect
[2024-03-26 09:26:16.610610] [gdk:info] change_state_to: requesting state disconnected
[2024-03-26 09:26:16.610610] [gdk:info] net: desired disconnected actual connected
[2024-03-26 09:26:16.610610] [gdk:info] net: disconnecting
```

**Comment by borgbyte** [comment]
There's a compatibility issue between jade 1.0.28 and green. You can:
 - downgrade to firmware 1.0.27
 - wait for firmware 1.0.29
 - wait for green 2.0.3

**Comment by delta1** [comment]
confirmed: works fine on jade firmware 1.0.27 and 1.0.29

**Comment by nakoshi-satamoto** [comment]
Very nice! Issue is resolved with upgrade 1.0.29 I am confirming also. That version was not showing up as an option in Green the other day but it is showing as a firmware upgrade now. I upgraded to version 1.0.29 and tested again. Great work to all the devs.
